src/workspaces/NewWorkspaceModal/NewWorkspaceModal.ts

```diff
@@ -284,6 +284,13 @@ const NewWorkspaceModal = withDisplayName(
             setNamespace(_.some({ projectName: namespace }, projects) ? namespace : undefined);
           }),
         Ajax(signal).Groups.list().then(setAllGroups),
+        !!cloneWorkspace &&
+          Ajax(signal)
+            .Workspaces.workspace(namespace, cloneWorkspace.workspace.name)
+            .details(['workspace.attributes.description'])
+            .then((workspace) => {
+              setDescription(workspace.workspace.attributes.description || '');
+            }),
```

Comment:
optional: is it worth checking here if the current workspace description is < 250 chars? If so, you could assume it wasn't truncated, and skip this call. (if you do this, might wanna make the "250" a constant somewhere)

Reply:
I think that would be ok _if_ `useWorkspaces` was guaranteed to only use that limit (like if it took a boolean parameter to limit string attribute length instead of a numeric max length). As is, that seems brittle, as some other place in the app _could_ call `useWorkspaces` with a different limit.